viewing_party/party.py

```diff
@@ -0,0 +1,183 @@
+# Wave One 
+# Part One 
+from statistics import mode 
+
+def create_movie(title, genre, rating):
+    if not title or not genre or not rating:
+        return None
+    else:
+        movie_list = {
+                "title" : title, 
+                "genre" : genre, 
+                "rating" : rating 
+        }
+        return movie_list
+        
+                                # wave 1 part 2
+def add_to_watched(user_data,movie):
+    watched_list = user_data["watched"]
+    watched_list.append(movie)
+    #user_data["watched"].append(movie)
+    return user_data
+                                # wave 1 part 3 
+def add_to_watchlist(user_data, movie):
+    add_list = user_data["watchlist"]
+    add_list.append(movie)
+    #user_data["watchlist"].append(movie)
+    return user_data
+                               # wave 1 part 4 
+def watch_movie(user_data,title): 
+    for i,d in enumerate(user_data['watchlist']):
+       if d['title'] == title:
+          user_data['watched'].append(d)
+          del user_data['watchlist'][i]      
```

Comment:
nice approach! another thing we could do is:

```suggestion
    for movie in user_data['watchlist']:
       if movie['title'] == title:
          user_data['watched'].append(movie)
          user_data['watchlist'].remove(movie)
```

this saves us from having to use a built-in function which I think simply turns the list of dictionaries into a tuple of dictionaries